src/runtime/manifest.ts

```diff
@@ -475,7 +484,7 @@ ${e.message}
       // Loading of imported manifests is triggered in parallel to avoid a serial loading
       // of resources over the network.
       await Promise.all(items.map(async item => {
-        if (item.kind === 'import') {
+        if (item.kind === 'import' && loader) {
```

Comment:
is `loader` ever null? maybe this should be an assert?

Reply:
Some tests are not supplying a loader.  It does seem reasonable to throw when you are attempting to process a manifest with an import and without a loader.